lib/mobius/cog.ex

```diff
@@ -228,7 +228,12 @@ defmodule Mobius.Cog do
   myBot: Invalid type for argument "num2". Expected "integer", got "hello".
   """
   defmacro command(command_name, context, args, do: block) do
-    # TODO: assert that the command name contains no whitespace
+    if not Regex.match?(~r/^[a-z0-9_]+$/, command_name) do
```

Comment:
```suggestion
    if not Regex.match?(~r/^[[:lower:][:digit:]_]+$/, command_name) do
```

I don't have a strong opinion on readability here, but [character classes](https://en.wikipedia.org/wiki/Regular_expression#Character_classes) are more versatile.